.github/workflows/release-please.yml

```
@@ -0,0 +1,38 @@
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+      with:
+        fetch-depth: 0
+
+    - name: Run Release Please Action
+      uses: googleapis/release-please-action@16a9c90856f42705d54a6fda1823352bdc62cf38 # v4.4.0
+      id: release
+      with:
+        token: ${{ steps.generate-token.outputs.token }}
```

actionlint (1.7.12)

[error] property "generate-token" is not defined in object type {}

(expression)